src/libYARP_dev/src/modules/ControlBoardWrapper/ControlBoardWrapper.h

```diff
@@ -347,6 +348,8 @@ class yarp::dev::ControlBoardWrapper:   public yarp::dev::DeviceDriver,
     bool ownDevices;
 #endif  //DOXYGEN_SHOULD_SKIP_THIS
 
+    bool threadInit();
```

Comment:
Adding a method to a class or changes the ABI, therefore according to the new workflow, this should not be done on the `master` branch.
As discussed with @barbalberto, next version of the patch will fix this issue without changing the ABI, and this patch will be applied on devel later, after this pr is merged.